languages/common-lisp/reference/exercise-concepts/word-count.md

```diff
@@ -0,0 +1,39 @@
+# General Concepts of Word Count
```

Comment:
Reviewing this I see I need to do a better job in my PR! This is good and detailed!